tests/lib/linter/linter.js

```diff
@@ -1846,7 +1846,7 @@ describe("Linter", () => {
             linter.defineRule(code, {
                 create: context => ({
                     Literal(node) {
-                        context.report(node, context.getPhysicalFilename());
+                        context.report(node, context.physicalFilename);
```

Comment:
Similar to https://github.com/eslint/eslint/pull/17108#discussion_r1176205663, I think we should keep testing `context.getPhysicalFilename()`.

```suggestion
                        assert.strictEqual(context.getPhysicalFilename(), context.physicalFilename);
                        context.report(node, context.physicalFilename);
```

(the same for other places where we switched to testing `context.physicalFilename`)

Reply:
We do have dedicated test cases for `context.getPhysicalFilename()`, do we still need this here?

https://github.com/eslint/eslint/blob/1aaece943ba73987344fcd8941109d4484335347/tests/lib/linter/linter.js#L9112

Reply:
It's only one test and it's under "Linter with FlatConfigArray", so we wouldn't have any tests with `context.getPhysicalFilename()` in eslintrc mode, which is what's currently used in production.

Reply:
@mdjermanovic Updated 👍🏻